eng/pipelines/templates/jobs/ci.yml

```diff
@@ -151,6 +151,7 @@ jobs:
               Paths:
                 - "/*"
                 - "!SessionRecords"
+      - template: /eng/common/pipelines/templates/steps/validate-filename.yml
```

Comment:
Have you done any testing of this when using sparse checkout? Probably worth validating the git commands we are using work correctly in this case.